ExtraDry/Sample.Shared/Entitites/Company.cs

```diff
@@ -14,7 +14,7 @@ public enum OwnershipStructure
 [Format(Icon = "company")]
 [FactTable, DimensionTable]
 [DeleteRule(DeleteAction.Recycle, nameof(Status), CompanyStatus.Deleted, CompanyStatus.Active)]
-public class Company : IResourceIdentifiers {
+public class Company : IResourceIdentifiers, IViewModelDisplayName {
```

Comment:
I was thinking that this would be on the ViewModel not on the Model. That would allow a ViewModel to override any property that was declared on the Model with its own view-model representation. When done on the Model, there is already a string Title on the IResourceIdentifiers that could be used.